tsconfig.json

```diff
@@ -18,7 +18,8 @@
     },
     "pretty": true,
     "noEmit": true,
-    "declaration": false,
+    "declaration": true,
```

Comment:
I think these 2 should just be in the tsconfig/ directory files. The top-level config (this file) is meant just for type checking and the editor to use.